src/server/ServerHandler.java

```diff
@@ -13,11 +15,19 @@
  * login, registration, game invitations, and gameplay moves.
  */
 public class ServerHandler extends Thread {
+
+    private Socket socket;           // Stores the client connection
+    private String currentUsername;  // Stores the client's username
```

Comment:
For both fields, use the typical Javadoc comment format and make both fields `final`
 
```suggestion
    /**
     * Stores the client connection.
     */
    private final Socket socket;

    /**
     * Stores the client's username.
     */
    private final String currentUsername;
```